frontend/src/metabase/visualizations/shared/settings/series.ts

```diff
@@ -0,0 +1,57 @@
+import _ from "underscore";
+import { getIn } from "icepick";
+import type { VisualizationSettings } from "metabase-types/api";
+import { getColorsForValues } from "metabase/lib/colors/charts";
+import type { ComputedVisualizationSettings } from "metabase/visualizations/types";
+
+export const SETTING_ID = "series_settings";
```

Comment:
```suggestion
export const SERIES_SETTING_KEY = "series_settings";
```

I think the name should explicitly include `SERIES` since it's for series settings, and it should be `KEY` since it's an object key.